### ISSUE TITLE
getAttr causes unexpected errors

### ISSUE THREAD

**Issue by cies** [faylang/fay-jquery#28]
When the following code gets executed using fay-19.1.2, fay-base-0.19.1.1, fay-text-0.3.0.1 and fay-jquery-0.6.0.2 (in a Yesod project), I get some JS errors that I dont expect and it stops executing.

``` haskell
showMenuSection :: Event -> Fay ()
showMenuSection _ = do
    ww <- select "#menu-nav-texts"
    alert ww                                                                                   
    w <- getAttr "id" ww  -- this seems to fail
    alert "woo"  -- this never gets executed
```

I have the following error message in my Chrome console, and the last 2 are showing up only after I close the first alert box.

```
Uncaught ReferenceError: Home is not defined autogen-TVvntntj.js:9
Uncaught TypeError: Cannot read property 'length' of undefined fay-runtime.js?9jSeXj7b:350
event.returnValue is deprecated. Please use the standard event.preventDefault() instead. jquery.js?etag=by8V3hgu:5374
Uncaught TypeError: Cannot read property 'length' of undefined fay-runtime.js?9jSeXj7b:350
Uncaught ReferenceError: Fay is not defined fay-runtime.js?9jSeXj7b:370
Uncaught TypeError: Cannot read property 'length' of undefined fay-runtime.js?9jSeXj7b:350
```

I really have no idea what may be the cause of this. I'll try to investigate (as I've already done for a few hours; including updating to the latest and greatest), if I happen to find the answer I'll post it here...

Thanks for reading this far.  Have a nice day!
